plugins/frontend/backstage-plugin-argo-cd/src/plugin.ts

```diff
@@ -18,8 +19,19 @@ export const argocdPlugin = createPlugin({
   apis: [
     createApiFactory({
       api: argoCDApiRef,
-      deps: { discoveryApi: discoveryApiRef, identityApi: identityApiRef },
-      factory: ({ discoveryApi, identityApi }) => new ArgoCDApiClient({ discoveryApi, identityApi }),
+      deps: {
+        discoveryApi: discoveryApiRef,
+        identityApi: identityApiRef,
+        configApi: configApiRef,
+      },
+      factory: ({ discoveryApi, identityApi, configApi }) =>
+        new ArgoCDApiClient({
+          discoveryApi,
+          identityApi,
+          backendBaseUrl: configApi.getString('backend.baseUrl'),
+          searchInstances: configApi
```

Comment:
Nice. I kicked off the CI process now and will take another look tomorrow morning at these to get this over the line.

Reply:
We'll make a note to write better automation/documentation for the local runs of Cypress :+1:

Reply:
👍🏻 it could be me. It seems that the cmd is `yarn dev:ci` and it's running. Every once in a while I see lots of activity, but it doesn't seem to go anywhere? It's been running for quite a while.